product-catalog/conf/Products.routes

```diff
@@ -0,0 +1,2 @@
+
+GET     /       products.controllers.ProductController.index()
```

Comment:
rename to  `product-catalog/conf/products.routes` (lowercase product)
and the error message
```
[error] /home/michael/dev/sphere-sunrise/conf/routes:7: object Routes is not a member of package products
[error] ->          /products                 products.Routes
[error] /home/michael/dev/sphere-sunrise/conf/routes:7: object Routes is not a member of package products
[error] ->          /products                 products.Routes
[error] /home/michael/dev/sphere-sunrise/conf/routes:7: not found: value handler
[error] ->          /products                 products.Routes
[error] three errors found
```

will go away